ansible/roles/deploy-plugins/tasks/azure.yml

```diff
@@ -0,0 +1,53 @@
+- name: rename env_domain in preview_cdn.html for CDN
+  shell: |
+    echo "{{sunbird_portal_preview_cdn_url}}"
+    sed -i 's|cdn_url|{{sunbird_portal_preview_cdn_url}}|g' "{{currentws}}"/ansible/preview/preview_cdn.html
+  when: sunbird_portal_preview_cdn_url is defined
+  tags:
+    - preview
+
+- set_fact:
+    full_url: "{{ folder_name }}{{sunbird_public_storage_account_sas}}"
+  tags:
+    - content-editor
+    - collection-editor
+    - generic-editor
+    - preview
+
+- name: delete batch
+  shell: |
+    azcopy rm "https://{{ sunbird_public_storage_account_name }}.blob.core.windows.net/{{ plugin_container_name }}/{{ folder_name }}{{sunbird_artifact_storage_account_sas}}" --recursive=true
+  async: 3600
+  poll: 10
+  tags:
+    - content-editor
+    - collection-editor
+    - generic-editor
+    - preview
+
+- name: upload batch
+  command: "az storage blob upload-batch --destination {{ plugin_container_name }}/{{ folder_name }} --source {{ source_name }}"
+  async: 3600
+  poll: 10
+  tags:
+    - content-editor 
+    - collection-editor
+    - generic-editor
+    - preview
+    - editor
+    - core-plugins
+  
+
+- name: upload file
+  command: "az storage blob upload --container-name {{ plugin_container_name }} --file {{ source_file_name }} --name artefacts/content-player/content-player-{{ player_version_number }}.zip"
+  async: 3600
+  poll: 10  
+  tags:
+    - preview
+
+- name:  run az_copy.sh
+  shell: "bash {{ az_file_path }} {{ plugin_container_name }} {{ source_file }}"
```

Comment:
same as the above file path hardcoding

Reply:
file path cannot be hardcoded, it is in jenkins jobs workspace, so we are passing file path as extra args
ex: ${currentWs}/ansible/content-plugins
@rjshrjndrn

Reply:
```
    bash {{ az_file_path | d('content-plugins/test.sh') }} {{ plugin_container_name }} {{ source_file }}
```